custom/icds_reports/management/commands/sql_scripts/ap_data_pull_1.sql

```diff
@@ -0,0 +1,28 @@
+COPY(SELECT
+    awc.state_name,
+    awc.district_name,
+    awc.block_name,
+    awc.supervisor_name,
+    awc.awc_name,
+    awc.awc_site_code,
+    CASE WHEN (SUM(CASE WHEN chm.recorded_weight IS NOT NULL THEN 1 ELSE 0 END)/count(*))::float>=0.6 THEN 'Y'
```

Comment:
@Rohit25negi Is it good to do the calculation here, like is there any chance of divide by 0 case here, if that's how this code gonna handle?